### PR TITLE
Rename ScreenClearing functions

### PR DESCRIPTION
**Pull request by tomlister**
#1315

### CODE REVIEW

File: examples/windowsize/main.go [hunk 1]
```diff
@@ -143,7 +143,7 @@ func (g *game) Update(screen *ebiten.Image) error {
 	g.transparent = ebiten.IsScreenTransparent()
 	floating := ebiten.IsWindowFloating()
 	resizable := ebiten.IsWindowResizable()
-	clearingScreenSkipped := ebiten.IsClearingScreenSkipped()
+	isScreenCleared := ebiten.IsScreenClearedEveryFrame()
```
Comment (hajimehoshi) [inline examples/windowsize/main.go:146]: I prefer `screenCleared` as a local variable here for consistency

File: uicontext.go [hunk 2]
```diff
@@ -176,7 +176,7 @@ func (c *uiContext) updateOffscreen() {
 	}
 	if c.offscreen == nil {
 		c.offscreen = newImage(sw, sh, FilterDefault)
-		c.offscreen.mipmap.SetVolatile(!IsClearingScreenSkipped())
+		c.offscreen.mipmap.SetVolatile(!IsScreenClearedEveryFrame())
```
Comment (hajimehoshi) [inline uicontext.go:179]: Remove `!`. The meaning is changed.

File: uicontext.go [hunk 3]
```diff
@@ -300,7 +300,7 @@ func (c *uiContext) update() error {
 
 		// Multiple successive Clear call should be integrated into one graphics command, then
 		// calling Clear on every Update should not affect the performance.
-		if !IsClearingScreenSkipped() {
+		if !IsScreenClearedEveryFrame() {
```
Comment (hajimehoshi) [inline uicontext.go:303]: ditto

File: uicontext.go [hunk 4]
```diff
@@ -318,7 +318,7 @@ func (c *uiContext) draw() {
 	}
 
 	if game, ok := c.game.(interface{ Draw(*Image) }); ok {
-		if !IsClearingScreenSkipped() {
+		if !IsScreenClearedEveryFrame() {
```
Comment (hajimehoshi) [inline uicontext.go:321]: ditto

File: uicontext.go [hunk 5]
```diff
@@ -193,7 +193,7 @@ func (c *uiContext) updateOffscreen() {
 	// scale. This is fine since ebiten.ScreenScale will be deprecated.
 }
 
-func (c *uiContext) setClearingScreenSkipped(skipped bool) {
+func (c *uiContext) setScreenClearedEveryFrame(skipped bool) {
```
Comment (hajimehoshi) [inline uicontext.go:196]: Rename `skipped`.

**Review by hajimehoshi — CHANGES_REQUESTED** [review]

File: run.go [hunk 6]
```diff
@@ -90,7 +90,7 @@ func CurrentFPS() float64 {
 
 var (
 	isDrawingSkipped        = int32(0)
-	isClearingScreenSkipped = int32(0)
+	isScreenClearingSkipped = int32(0)
```
Comment (hajimehoshi) [inline run.go:93]: The default value is `int32(1)`.

Comment (hajimehoshi) [reply run.go:93]: Also please rename this variable to `isScreenClearedEveryFrame`

File: uicontext.go (hunk 5 above)
Comment (hajimehoshi) [reply uicontext.go:196]: Not updated yet?

Comment (tomlister) [reply uicontext.go:196]: I `skipped` this one by accident my bad.

File: uicontext.go [hunk 7]
```diff
@@ -193,12 +193,12 @@ func (c *uiContext) updateOffscreen() {
...
 }
 
-func (c *uiContext) setClearingScreenSkipped(skipped bool) {
+func (c *uiContext) setScreenClearedEveryFrame(cleared bool) {
 	c.m.Lock()
 	defer c.m.Unlock()
 
 	if c.offscreen != nil {
-		c.offscreen.mipmap.SetVolatile(!skipped)
+		c.offscreen.mipmap.SetVolatile(!cleared)
```
Comment (hajimehoshi) [inline uicontext.go:201]: The meaning is flipped. Remove `!`

**Review by hajimehoshi — APPROVED** [review]
LGTM, thank you!

**Pull request merged**